diesel_cli/src/pretty_printing.rs

```diff
@@ -82,7 +82,7 @@ pub fn format_schema(schema: &str) -> Result<String, FmtError> {
         }
     }
 
-    Ok(out.replace("\t", "    "))
+    Ok(out.replace("\t", "    ").replace("table!", "\ntable!").trim().to_string())
```

Comment:
I wonder if there's a pull-based string processor crate. Like an iterator, but with replace and trim and such